websites/P/Padlet/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "padlet",
+  "author": {
+    "name": "The Gamerzs",
+    "id": "730142125181894657"
+  },
+  "category": "other",
+  "logo": "https://i.imgur.com/sxpYxcR.png",
+  "thumbnail": "https://i.imgur.com/9nXPXZh.png",
```

Comment:
Thumbnail should be a screenshot of the homepage or a promotional banner, neither of which this meets